public/app/app.jsx

```diff
@@ -1,32 +1,27 @@
 var React = require('react');
-var Reflux = require("reflux");
-
+require('reflux');
 
 // if style! is used here, it will attach the style to DOM automatically
 require('style!./project-setup/all.scss');
 
-var actions = require("./actions");
+require('./actions');
 
 var Router = require('react-router');
-var RouteHandler = Router.RouteHandler;
 var Route = Router.Route;
-var DefaultRoute = Router.DefaultRoute;
-var NotFoundRoute = Router.NotFoundRoute;
 var Redirect = Router.Redirect;
-var Link = Router.Link;
 
 var MainScreen = require('./screens/main');
 var DevicesRegion = require('./screens/main/sections/devices');
 var SettingsRegion = require('./screens/main/sections/settings');
 
 var routes = (
-  <Route path="/" handler={MainScreen}>
-    <Route name="settings" handler={SettingsRegion}/>
-    <Route name="devices" handler={DevicesRegion}/>
-    <Redirect from="/" to="devices"/>
+  <Route path='/' handler={MainScreen}>
```

Comment:
I'd say syntax highlighting is quite sufficient for distinguishing purposes. Personally I have nothing against double quotes in markup, but there seems to be no way to configure ESLint to distinguish between js and jsx.